setup.py

```diff
@@ -2,11 +2,6 @@
 
 setup(
     name="indexd",
```

Comment:
So we are not using setuptools_scm?
Wouldn't that just leave us maintaining our own "version" generation piece?
I know you mentioned that setuptools_scm didn't quite get us there but maybe we are still just missing an important piece of understanding so setuptools_scm does the thing it's designed for?

Reply:
we are still using setuptools_scm